src/main/resources/User_info.json

```diff
@@ -0,0 +1,5 @@
+{
+  "firstName": "Sergey",
+  "lastName": "Prokofiev",
+  "age": 29
+}
```

Comment:
What's the reason to have this resource? Do you use it from code? Or it's just template for message?